01_dask.delayed.ipynb

```diff
@@ -31,7 +39,7 @@
    "source": [
     "from dask.distributed import Client\n",
     "\n",
-    "client = Client()"
+    "client = Client(n_workers=4)"
```

Comment:
On binder, we get 1 CPU core. I hardcoded this to 4, which is probably fine for the tutorial even if it isn't optimal for everyone's machine.